src/js/components/ServerLink/ServerLink.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { mount } from 'enzyme';
+import ServerLink from "../ServerLink/ServerLink";
+import { BrowserRouter } from "react-router-dom";
+
+describe('ServerLink', () => {
+  it('displays an active state', () => {
+    const wrapper = mount(
+      <BrowserRouter>
+        <ServerLink id={ 'foo' } />
+      </BrowserRouter>
+    );
+
+    expect(wrapper.find('NavLink').prop('activeClassName')).toEqual('nav-channel-active');
```

Comment:
Yeah... I guess this is proving that it has a prop, and we should trust that the `NavLink` logic is tested by the vendor. But this sort of only tests a somewhat static value in the code. Is there a way to test if the className is applied?